app/lint-baseline.xml

```diff
@@ -1660,7 +1506,7 @@
 
     <issue
         id="IconXmlAndPng"
-        message="The following images appear both as density independent `.xml` files and as bitmap files: /Users/oracle/Projects/fenix/app/src/main/res/drawable-hdpi/ic_logo_wordmark_normal.png, /Users/oracle/Projects/fenix/app/src/main/res/drawable-night/ic_logo_wordmark_normal.xml">
+        message="The following images appear both as density independent `.xml` files and as bitmap files: /Users/rotbolt/AndroidStudioProjects/fenix/app/src/main/res/drawable-hdpi/ic_logo_wordmark_normal.png, /Users/rotbolt/AndroidStudioProjects/fenix/app/src/main/res/drawable-night/ic_logo_wordmark_normal.xml">
```

Comment:
this doesn't look correct as it's pointing to your local path.

Reply:
If you look at the previous statement, even that was pointing to local path

Reply:
For now, let me remove this. But I think lint would generate this local path only
when re-generating this file, committer needs to take care for removing the local path

Reply:
I have updated to have relative path from fenix folder now